### DISCUSSION TITLE
Other Asset Types

### DISCUSSION THREAD

**Discussion by rgstephens** [ghostfolio/ghostfolio#876] [Ideas]
Is there interest in adding features to support other asset types?  Real Estate, Angel/Privately held investments, Precious Metals?

**Comment by dtslvr** [comment]
Hi @rgstephens

Yes, definitely.

- [x] Real estate and private equity (e.g. angel investments) can be recorded via (wealth) items with the [next version](https://github.com/ghostfolio/ghostfolio/blame/main/CHANGELOG.md#L10).
- [x] Precious metals are supported since [Ghostfolio 1.144.0](https://github.com/ghostfolio/ghostfolio/releases/tag/1.144.0)
- [ ] To display (wealth) items in the portfolio allocations by asset class, an adjustment is still required

Did you have something like that in mind? I would be happy to get your feedback.

**Reply by rgstephens** [reply]
Thanks, having trouble finding Futures to add metals (maybe that's Premium).  For precious metals, I'd like to add physical metal types and create an account name related to where those physical metals are stored.

**Reply by dtslvr** [reply]
Please add an activity and search for _Gold_. You can track your fraction and link your account there. I hope this helps 🙂 

<img width="1020" alt="precious-metal-activity" src="https://user-images.githubusercontent.com/4159106/166135425-658b494c-a0ee-4a25-b4b5-0715cbc3eabc.png">

**Reply by lovefamilymain** [reply]
Hello, unfortunatly I have no "Gold" in the list.

**Reply by stefan786** [reply]
Same here, no "Gold" in the list, even though I added GC=F as an asset.

**Comment by sfuerte** [comment]
@dtslvr, trying to understand if I missed something:
- I can create an account and add activities for `SI=F` or `GC=F` on the required dates, and it'll be all tracked;
- but then there is `Type: Valuable`, `Class: Commodity`, `Sub Class: Precious Metal`, which would be just sitting with assigned "value" and no tracking in metal price changes, am I missing something for its purpose? 

![image](https://github.com/user-attachments/assets/c5c4217e-13e7-4b1f-b9de-4dcdd39a6bf2)
